merge_request_approvals.go

```diff
@@ -33,6 +33,7 @@ type MergeRequestApprovals struct {
 	ApprovedBy           []*MergeRequestApproverUser  `json:"approved_by"`
 	Approvers            []*MergeRequestApproverUser  `json:"approvers"`
 	ApproverGroups       []*MergeRequestApproverGroup `json:"approver_groups"`
+	SuggestedApprovers   []*ApproverUser              `json:"suggested_approvers"`
```

Comment:
I cannot find this in the docs, can you point me to a location that shows this is indeed a valid field?

Reply:
There is no information about this parameter in docs. But it works. Version GitLab Enterprise Edition 12.2.5-ee

Reply:
I need to see some reference to this in order to be able to judge if this is the best way to add this field. A link to the definition in the Gitlab source code would also be fine.

Reply:
Cf. https://gitlab.com/gitlab-org/gitlab/blob/master/ee/lib/ee/api/entities.rb#L425-428